### ISSUE TITLE
CI: k8s-1.8.K8sFQDNTest Restart Cilium validate that FQDN is still working

### ISSUE THREAD

**Issue by brb** [cilium/cilium#6629]
```
/home/jenkins/workspace/cilium-ginkgo_cilium_master/src/github.com/cilium/cilium/test/ginkgo-ext/scopes.go:408
"app2" can  connect when it should not work
Expected command: kubectl exec -n default app2-cd69fd9f6-66vkl -- curl -s -D /dev/stderr --fail --connect-timeout 3 --max-time 8 192.168.9.11 -w "time-> DNS: '%{time_namelookup}(%{remote_ip})', Connect: '%{time_connect}',Transfer '%{time_starttransfer}', total '%{time_total}'"
To have failed, but it was successful:
Exitcode: 0 
Stdout:
 	 <html><body><h1>It works!</h1></body></html>
	 time-> DNS: '0.000020(192.168.9.11)', Connect: '0.000155',Transfer '0.000798', total '0.001588'
Stderr:
 	 HTTP/1.1 200 OK
	 Date: Mon, 14 Jan 2019 00:52:15 GMT
	 Server: Apache/2.4.25 (Unix)
	 Last-Modified: Mon, 11 Jun 2007 18:53:14 GMT
	 ETag: "2d-432a5e4a73a80"
	 Accept-Ranges: bytes
	 Content-Length: 45
	 Content-Type: text/html
	 
	 

/home/jenkins/workspace/cilium-ginkgo_cilium_master/src/github.com/cilium/cilium/test/k8sT/fqdn.go:180
```

https://jenkins.cilium.io/job/cilium-ginkgo/job/cilium/job/master/2131/testReport/junit/k8s-1/8/K8sFQDNTest_Restart_Cilium_validate_that_FQDN_is_still_working/

[0976d448_K8sFQDNTest_Restart_Cilium_validate_that_FQDN_is_still_working.zip](https://github.com/cilium/cilium/files/2754624/0976d448_K8sFQDNTest_Restart_Cilium_validate_that_FQDN_is_still_working.zip)

/cc @eloycoto @raybejjani

**Comment by tgraf** [comment]
Happened with the 5 sec connect timeout as well but it seems a lot less frequent
```
Expected command: kubectl exec -n default app2-5b49d5dd9-8ng7t -- curl -s -D /dev/stderr --fail --connect-timeout 5 --max-time 8 192.168.9.10 -w "time-> DNS: '%{time_namelookup}(%{remote_ip})', Connect: '%{time_connect}',Transfer '%{time_starttransfer}', total '%{time_total}'" 
To succeed, but it failed:
Exitcode: 28 
Stdout:
 	 time-> DNS: '0.000022()', Connect: '0.000000',Transfer '0.000000', total '5.003070'
Stderr:
 	 command terminated with exit code 28
```

https://jenkins.cilium.io/job/cilium-ginkgo/job/cilium/job/master/2221/testReport/k8s-1/13/K8sFQDNTest_Restart_Cilium_validate_that_FQDN_is_still_working/

**Comment by eloycoto** [comment]
No clue what could happen here:

The Endpoint is mapped correctly to a identity `16777217`

If you see the policy for the endpoint all is allowed:

```
/sys/fs/bpf/tc/globals/cilium_policy_1823:

DIRECTION   IDENTITY   PORT/PROTO   PROXY PORT   BYTES   PACKETS
Ingress     1          ANY          NONE         0       0
Ingress     2          ANY          NONE         0       0
Ingress     3          ANY          NONE         0       0
Ingress     4          ANY          NONE         0       0
Ingress     5          ANY          NONE         0       0
Ingress     100        ANY          NONE         0       0
Ingress     101        ANY          NONE         0       0
Ingress     102        ANY          NONE         0       0
Ingress     103        ANY          NONE         0       0
Ingress     104        ANY          NONE         0       0
Ingress     105        ANY          NONE         0       0
Ingress     106        ANY          NONE         0       0
Ingress     4251       ANY          NONE         0       0
Ingress     10519      ANY          NONE         0       0
Ingress     25995      ANY          NONE         0       0
Ingress     37281      ANY          NONE         0       0
Ingress     16777217   ANY          NONE         0       0
Egress      1          53/TCP       44869        0       0
Egress      1          53/UDP       44869        0       0
Egress      2          53/TCP       44869        0       0
Egress      2          53/UDP       44869        0       0
Egress      3          53/UDP       44869        0       0
Egress      3          53/TCP       44869        0       0
Egress      4          53/UDP       44869        0       0
Egress      4          53/TCP       44869        0       0
Egress      5          53/TCP       44869        0       0
Egress      5          53/UDP       44869        0       0
Egress      100        53/UDP       44869        0       0
Egress      100        53/TCP       44869        0       0
Egress      101        53/UDP       44869        0       0
Egress      101        53/TCP       44869        0       0
Egress      102        53/TCP       44869        0       0
Egress      102        53/UDP       44869        0       0
Egress      103        53/TCP       44869        0       0
Egress      103        53/UDP       44869        0       0
Egress      104        53/TCP       44869        0       0
Egress      104        53/UDP       44869        0       0
Egress      105        53/TCP       44869        0       0
Egress      105        53/UDP       44869        0       0
Egress      106        53/TCP       44869        0       0
Egress      106        53/UDP       44869        0       0
Egress      4251       53/TCP       44869        0       0
Egress      4251       53/UDP       44869        0       0
Egress      10519      53/UDP       44869        0       0
Egress      10519      53/TCP       44869        0       0
Egress      25995      53/TCP       44869        0       0
Egress      25995      53/UDP       44869        0       0
Egress      37281      53/TCP       44869        0       0
Egress      37281      53/UDP       44869        0       0
Egress      16777217   ANY          NONE         0       0
Egress      16777217   53/UDP       44869        0       0
Egress      16777217   53/TCP       44869        0       0
```

Cilium has been restored, so there is no logs or any monitor information. @joestringer  any idea here?

**Comment by tgraf** [comment]
@eloycoto It is possible that it is not policy related at all but a connectivity problem

**Comment by tgraf** [comment]
@brb How often did you see this?

**Comment by tgraf** [comment]
Likely a CI bug in that the CI continues when the CNP status field is missing

**Comment by brb** [comment]
> How often did you see this?

Saw it only once.

**Comment by ianvernon** [comment]
This is occurring much more frequently in the CI now after #6740 was merged. @tgraf filed #6957 , which may overlap with the work on this ticket.

**Comment by eloycoto** [comment]
close due a new issue has been open.
